### PR TITLE
Vanilla empty state

### PR DESCRIPTION
**Pull request by yoshuawuyts**
- moved the table element from the main view to `elements/table.js`
- created an empty state for the main view that's shown if there are no elements in the thing - meant to be styled by @kriesse as per #68 
- use the `prettier-bytes` package to make our byte display a bit less noisy
- "delete a dat" is broken as per https://github.com/datproject/dat-desktop/pull/91 - fixed some of the symptoms but it's still not working - opened an issue for it here https://github.com/datproject/dat-desktop/issues/104

Thanks!

### CODE REVIEW

File: elements/table.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+const encoding = require('dat-encoding')
+const bytes = require('prettier-bytes')
+const html = require('choo/html')
+
+const button = require('../elements/button')
```
Comment (juliangruber) [inline elements/table.js]: can be simplified to `require('./button')`

File: lib/dat-manager.js [hunk 2]
```diff
@@ -56,21 +56,15 @@ module.exports = class Manager extends EventEmitter {
     })
   }
   remove (dat, cb) {
+    const self = this
     dat.listStream.destroy()
-    dat.close(err => {
+    dat.close(function (err) {
```
Comment (juliangruber) [inline lib/dat-manager.js]: woot?

Comment (yoshuawuyts) [inline lib/dat-manager.js]: oh lol

File: elements/table.js (hunk 1 above)
Comment (yoshuawuyts) [inline elements/table.js]: good point!

**Comment by yoshuawuyts** [comment]
@Kriesse should be able to style it now :D

**Pull request merged**